civicase.php

```diff
@@ -443,8 +443,20 @@ function civicase_civicrm_postProcess($formName, &$form) {
   if (in_array($formName, $specialForms)) {
     $urlParams = parse_url(htmlspecialchars_decode($form->controller->_entryURL), PHP_URL_QUERY);
     parse_str($urlParams, $urlParams);
-    if (!empty($urlParams['draft_id'])) {
-      civicrm_api3('Activity', 'delete', ['id' => $urlParams['draft_id']]);
+
```

Comment:
Are we sure we can do away with this code? I think this deletes the draft activity right? Don't we still need to delete it at some point?

Reply:
Yes, it is not required anymore. Because deletion of Activities are done seprately. But here once we want to move the Activity from Draft to Completed. It should not delete the activity, instead it should just change its status.

Reply:
We should probably move this logic to it's own class but we can leave as it is for now as there are some improvements to the Draft functionality and we can do this then.